src/features/campaigns/hooks/useClusteredActivities.ts

```diff
@@ -157,3 +173,19 @@ export default function useClusteredActivities(
     return aStart.getTime() - bStart.getTime();
   });
 }
+
+function doesMultipleLocationEventsMatch(
+  event1: ZetkinEvent,
+  event2: ZetkinEvent
+): boolean {
+  return (
+    !!event1.location &&
+    !!event2.location &&
+    event1.location.id !== event2.location.id &&
+    event1.activity.id === event2?.activity.id &&
```

Comment:
why a ? after event2? Is it not certain at this point in the logic that event2 exists?